### ISSUE TITLE
Why do images do poorly when the resolution is less than 100

### ISSUE THREAD

**Issue by huangqianfirst** [tensorlayer/srgan#104]
I am confuse why some test image greater than 300pix can works well, but when i test images which resolution less than 100pix or around 100pixel , the texture detail can't recover while the model it trained by 9696 to 384384 . like below , lr is input, 4x_gen is output. I kown it lr images may has been lost many detil, but why result is so awful and how **can i impove upscale result of images resolution around 100pixel.
![3_4x_gen_1200_2040](https://user-images.githubusercontent.com/11473038/44195862-e8dae680-a16c-11e8-956a-549b5fdd8f25.png)
![3_lr_300_510](https://user-images.githubusercontent.com/11473038/44195864-ea0c1380-a16c-11e8-9a02-96dc6577d48e.png)
![sence6_4x_gen_400_748](https://user-images.githubusercontent.com/11473038/44195865-ebd5d700-a16c-11e8-8cf0-9b73c93ed9d4.png)
![sence6_lr_100_187](https://user-images.githubusercontent.com/11473038/44195866-ec6e6d80-a16c-11e8-8090-d3d5a6af2445.png)

**Comment by wangyifenggg** [comment]
I met the same problems. Maybe using a different dataset can solve it. @huangqianfirst

**Comment by huangqianfirst** [comment]
Yes . I trained with high resolution image and low resolution with JEPG noise images. This problem has been resolved.

**Comment by JunbinWang** [comment]
> @wangyifenggg . Yes . I trained with high resolution image and low resolution with JEPG noise images. This problem has been resolved.

hi @wangyifenggg  does the SRGAN model can handle the image compression artifacts by training it 
with jpeg LR dataset ?

**Comment by wangyifenggg** [comment]
> > @wangyifenggg . Yes . I trained with high resolution image and low resolution with JEPG noise images. This problem has been resolved.
> 
> hi @wangyifenggg does the SRGAN model can handle the image compression artifacts by training it
> with jpeg LR dataset ?

If you want to make your own training dataset, it's important to choose an appropriate way to make your LR dataset. For example, if you use nearest interpolation, the LR pictures will not smooth enough, which will damage the output of the SRGAN model. Maybe you can use area interpolation to produce LR pictures which are more smooth, and you can get better training result.
